OpenOversight/app/static/js/find_officer.js

```diff
@@ -37,9 +37,14 @@ $(document).ready(function() {
             data: {department_id: dept_id}
         }).done(function(ranks) {
             $('input#rank').replaceWith('<select class="form-control" id="rank" name="rank">');
+            var rank_box = $('select#rank')
+            // Add the null case first
+            rank_box.append(
+                $('<option></option>').attr("value", "Not Sure").text("Not Sure")
+            );
             for (i = 0; i < ranks.length; i++) {
                 console.log(ranks[i]);
```

Comment:
Can we remove this console.log?

Reply:
Oh yea, I meant to do that haha. Thanks!